packages/edit-navigation/src/components/editor/style.scss

```diff
@@ -8,7 +8,9 @@
 }
 
 .edit-navigation-editor__block-view {
+	padding: $grid-unit-20;
 	@include break-medium() {
+		height: 100%;
```

Comment:
This isn't working on Safari for some reason:

<img width="1159" alt="Screen Shot 2021-01-29 at 11 50 23 am" src="https://user-images.githubusercontent.com/8096000/106218363-470bbf00-622b-11eb-9ee7-a81da133b518.png">

Using `align-self: stretch` instead should fix it.

Reply:
Nice. Thanks for the suggestion (and finding that issue).